### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Authentication Pages
- login page
- sign-up page
Screenshot (45)
Screenshot (46)

## Responsiveness:
- Mobile[✔]
- Medium Screen[✔]
- Large Screen[✔]

### CODE REVIEW

File: Authentication/Login.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      href="https://readymade-ui.netlify.app/CSS/main.css"
+    />
+    <link rel="stylesheet" href="/Authentication/Authentication.css" />
+
+    <link rel="stylesheet" href="/css/main.css" />
+    <title>Login | GamersKart</title>
+  </head>
+  <body>
+    <section class="wrapper">
+      <nav class="navigation">
```
Comment (bhavyaj12) [inline Authentication/Login.html:27]: Login button in first page is too much to the corner, maybe give some margin

File: Authentication/Login.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                  <div>
+                    <input type="checkbox" name="remember-me" id="" />
+                    <label for="remember-me">Remember Me</label>
+                  </div>
+                  <span><a href="#"></a>Forgot Password</span>
+                </div>
+                <button class="btn btn-solid">Login</button>
+              </div>
+              <a class="sign-up" href="/Authentication/Signup.html">
+                Don't have account?Sign up!
```
Comment (bhavyaj12) [inline Authentication/Login.html]: Put space between 'account?sign up'

File: Authentication/Signup.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="description"
```
Comment (bhavyaj12) [inline Authentication/Signup.html:7]: Gamming spelling wrong

File: index.css [hunk 4]
```diff
@@ -34,7 +34,7 @@
 
 .home-banner {
   width: 100%;
-  aspect-ratio: 2/1;
+  /* aspect-ratio: 2/1; */
```
Comment (bhavyaj12) [inline index.css]: Removed unused comments

**Review by bhavyaj12 — COMMENTED** [review]
Added some comments, rest looks good to merge!

**Review by harshitpaliwal95 — APPROVED** [review]
looks good

**Pull request merged**